calico-cloud/image-assurance/scan-image-registries.mdx

```diff
@@ -248,34 +251,27 @@ To change the CVSS threshold values, note the following:
 
 When Runtime View is enabled in Scan Settings, {{prodname}} actively monitors pods running in your Kubernetes clusters so it can correlate them with images you have recorded in the Image Assurance system. By default, this feature is disabled. If you disable the feature after it has been enabled, runtime information that is already collected for existing images will remain in the system.
 
-### Create vulnerability exceptions
+### Exclude specific vulnerabilities
 
-In some cases, you may want to ignore specific vulnerabilities or CVEs. For example:
+In some cases, you may want to exclude specific vulnerabilities from your scan results. For example, you may determine that a warning is a false positive because of the way you are using an affected package. By creating a vulnerability exception, you can configure Image Assurance to ignore a CVE for a specific image, all images from the same repo, or for all images.
 
-- You determine scan results are "false positives" (based on how you are using the affected package)
-- You decide to exclude a vulnerability based on your own internal assessment
-
-To exclude a vulnerability from scanning results, create a vulnerability exception.
-
-In the left navbar in Manager, click **Image Assurance**, **Vulnerability Exceptions**.
-
-Before creating an exception, it is important to understand the effects to Image Assurance.
-
-- **Maximum CVSS score**
-
-  An image’s maximum CVSS score may be reduced to a lower score. An exception could eliminate the vulnerability with the highest CVSS score for an image.
-
-- **Scan results**
-
-  Scan results for Pass/Warn/Fail could change because they are based on the maximum CVSS score. For example, Fail could change to Warn, and Warn could change to Pass.
-
-- **Vulnerability global alerts**
-
-  Alerts may no longer be generated. Alerts are triggered based on scan results or maximum CVSS score values of images.
+:::caution 
+A vulnerability excpetion can significantly change how Image Assurance determines image health. 
+- **Maximum CVSS score** 
+An image’s maximum CVSS score may be reduced to a lower score. An exception could eliminate the vulnerability with the highest CVSS score for an image. 
+- **Scan results** 
+Scan results for Pass/Warn/Fail could change because they are based on the maximum CVSS score. For example, Fail could change to Warn, and Warn could change to Pass. 
+- **Vulnerability global alerts** 
+Alerts may no longer be generated. Alerts are triggered based on scan results or maximum CVSS score values of images. 
```

Comment:
This sentence may be fine if I was looking at the UI, but it isn't clear. Why would alerts no longer be generated? 

Alerts may no longer be generated.

Reply:
How about I change it to "Alerts may no longer be triggered."

The explanation is the second sentence that is there "Alerts are triggered based on scan results or maximum CVSS score values of images." 

So from the previous sections above this one, it explains that adding an exception maybe cause the maximum CVSS score to change, which in turn could make the scan result (Pass/Warn/Fail) to change for the affected image.

Reply:
I get it now. Both are ok. Rewrite is a little more precise.

Reply:
Okay, it's updated now.